.github/workflows/satellites.yml

```diff
@@ -1,170 +0,0 @@
-name: Test on Satellites
```

Comment:
All of the logic in this file is now in `ci-docker-ubuntu.yml` and `ci-podman-ubuntu.yml`